code/game/objects/structures/transit_tubes/transit_tube_pod.dm

```diff
@@ -54,8 +54,11 @@
 
 /obj/structure/transit_tube_pod/ex_act(severity, target)
 	. = ..()
-	if(!QDELETED(src))
-		empty_pod()
+	if(QDELETED(src))
+		return FALSE
```

Comment:
i think it's kinda weird because we aren't doing any actual work as far as deleting ourselves is concerned but sure
```suggestion
	if(QDELETED(src))
		return TRUE
```